poh/src/poh_service.rs

```diff
@@ -397,7 +398,11 @@ mod tests {
     #[ignore]
     fn test_poh_service() {
         solana_logger::setup();
-        let GenesisConfigInfo { genesis_config, .. } = create_genesis_config(2);
+        let GenesisConfigInfo {
+            mut genesis_config, ..
+        } = create_genesis_config(2);
+        let hashes_per_tick = Some(DEFAULT_HASHES_PER_TICK);
+        genesis_config.poh_config.hashes_per_tick = hashes_per_tick;
```

Comment:
need this so that `hashes_per_tick` gets set properly in the bank (and thus inherited properly by future banks, which gets inherited by PoH)